README.md

```diff
@@ -26,82 +26,18 @@ Basic example: https://subscription.streamlit.app
 
 I made st-paywall so data scientists and LLM developers can create small businesses around their Streamlit apps. Every week I see dozens of new incredible apps built in Streamlit that are adored by users, but eventually shut down or moved off of Streamlit as authentication and payment integration are too hard. This is my attempt at a dead-simple API around each, abstracting the both away into a single function (`add_auth`). Enjoy!
 
-## Overview
+## Documentation
 
-This package gives you one basic function (`add_auth`) that adds subscription functionality to your Streamlit apps. `add_auth` will add both a Google login button if they are not logged in, and a Stripe subscription button to your sidebar if they are not subscribed. If they are subscribed, `st.session_state.user_subscribed` will be true, and if they are logged in, `st.session_state.email` will have their email.
-If the `required` parameter is `True`, the app will stop with `st.stop()` if the user is not logged in and subscribed. Otherwise, you the developer will have control over exactly how you want to paywall the apps!
+Once you configure the authentication and subscription on `st.secrets`, you can use the the library methods to conditionally render the content of the page:
 
- I hope you use this to create tons of value, and capture some of it with the magic of Streamlit.
+```python
+import st-paywall as stw
 
-This package expects that you have a `.streamlit/secrets.toml` file which you will have to create. Inside it, you will need to add your Stripe and Google API information that runs the authentication and subscription parts of the package.
-
-### Stripe
-
-In order to set up your Stripe (the best and easiest payment infrastructure in the world), go to [Stripe.com](https://stripe.com) and make an account. Once you make an account, you need to make a [payment subscription link](https://dashboard.stripe.com/test/payment-links/create) (which is in test mode by default) and add the link to your app (currently held in streamlit_app.py). If the user to your app logs in and has not already signed up and paid via Stripe, they will be asked to subscribe before they can see the rest of the app.
-
-The subscription link should be added to secrets.toml like this.
-
-```toml
-stripe_link = 'https://buy.stripe.com/test_...'
+stw.user_is_logged_in()  # Boolean determining if the user is logged in or not. If not, redirects to subscription page
```

Comment:
it is stored in session state, so it is actually 

st.session_state.email and st.session_state.user_subscribed